python/tskit/trees.py

```diff
@@ -3823,19 +3848,24 @@ def num_migrations(self):
     @property
     def max_root_time(self):
         """
-        Returns time of the oldest root in any of the trees in this tree sequence.
+        Returns the time of the oldest root in any of the trees in this tree sequence.
         This is usually equal to ``np.max(ts.tables.nodes.time)`` but may not be
-        since there can be nodes that are not present in any tree. Consistent
-        with the definition of tree roots, if there are no edges in the tree
-        sequence we return the time of the oldest sample.
+        since there can be non-sample nodes that are not present in any tree. Note that
+        isolated samples are also defined as roots (so there can be a max_root_time
+        even in a tree sequence with no edges).
+
+        ..note::
```

Comment:
Missing a space here, ``.. note::``

Reply:
A ``raises ValueError`` would probably be the more systematic way of documenting this rather than a note block.

Reply:
Nice, good point